bin/apisix

```diff
@@ -128,13 +128,26 @@ stream {
     }
 
     server {
+        {% if enable_reuseport then %}
+
+        {% for _, port in ipairs(stream_proxy.tcp or {}) do %}
+        listen {*port*} reuseport;
+        {% end %}
+        {% for _, port in ipairs(stream_proxy.udp or {}) do %}
+        listen {*port*} udp reuseport;
+        {% end %}
+
+        {% else %} {% --if enable_reuseport %}
+
         {% for _, port in ipairs(stream_proxy.tcp or {}) do %}
         listen {*port*};
```

Comment:
`listen {*port*}  {% if enable_reuseport then %} reuseport {% end %};`

This style is simpler.